DNN Platform/Library/Services/Upgrade/Upgrade.cs

```diff
@@ -2785,8 +2785,6 @@ private static void UpgradeToVersion721()
 
                         if (userName != HttpUtility.HtmlDecode(userName))
                         {
-                            
-                            userName = HttpUtility.HtmlDecode(userName);

```

Comment:
i don't think this line can be removed, such as if the username is "&lt;testuser&gt;", if you don't decode it, the input filter can't remove the tags.